src/datasets/load.py

```diff
@@ -366,20 +365,19 @@ def infer_module_for_data_files(
         for ext, _ in extensions_counter.most_common():
             if ext in _EXTENSION_TO_MODULE:
                 return _EXTENSION_TO_MODULE[ext]
-            elif ext == ".zip":
-                return infer_module_for_data_files_in_archives(data_files_list, use_auth_token=use_auth_token)
+            elif ext == "zip":
+                return infer_module_for_data_files_in_archives(data_files_list, download_config=download_config)
```

Comment:
Maybe the CI fails comes from this, let me try applying this change

```suggestion
            elif ext == ".zip":
                return infer_module_for_data_files_in_archives(data_files_list, download_config=download_config)
```

Reply:
looks like that was it !

Reply:
great, thank you very much for helping me to find this issue. no worries, just take you time.